nipype/interfaces/io.py

```diff
@@ -1296,32 +1296,36 @@ class SelectFilesInputSpec(DynamicTraitedSpec, BaseInterfaceInputSpec):
 class SelectFiles(IOBase):
     """Flexibly collect data from disk to feed into workflows.
 
-    This interface uses the {}-based string formatting syntax to plug
+    This interface uses Python's {}-based string formatting syntax to plug
     values (possibly known only at workflow execution time) into string
-    templates and collect files from persistant storage. These templates
-    can also be combined with glob wildcards. The field names in the
-    formatting template (i.e. the terms in braces) will become inputs
-    fields on the interface, and the keys in the templates dictionary
-    will form the output fields.
+    templates and collect files from persistant storage. These templates can
+    also be combined with glob wildcards (*, ?) and character ranges ([...]).
```

Comment:
```suggestion
    also be combined with glob wildcards (``*``, ``?``) and character ranges (``[...]``).
```
If the * is not escaped (or decorated with an inline code block), sphinx will spit a warning and write everything following the \* in italics.